lms/static/sass/views/_login-register.scss

```diff
@@ -464,6 +465,25 @@ $sm-btn-linkedin: #0077b5;
             }
         }
 
+        &.button-oa2-azuread-oauth2 {
+            color: darken($sm-btn-azuread, 20%);
+
+            .icon {
+                background: $sm-btn-azuread;
+            }
+
+            &:hover,
+            &:focus {
+                background-color: $sm-btn-azuread;
+                border: 1px solid $sm-btn-azuread;
+                color: white;
```

Comment:
@omarkhan I think it would be better to use variable `$white` instead of `white`.